### PR TITLE
#42 editing/deleting columns

### PR DESCRIPTION
**Pull request by naazy**
#42 editing/deleting columns

### CODE REVIEW

File: src/MainUpdate.elm [hunk 1]
```diff
@@ -510,3 +512,93 @@ update msg model =
...
+            UpdatePickedTrack trackId trackInputType newPickedTrackInput ->
+                let
+                    pickedTrack =
+                        model.pickedTracks
+                            |> List.filter (\t -> t.id == trackId)
+                            |> List.head
+                            |> Maybe.withDefault (Track 0 "" "")
+
+                    newPickedTrack =
+                        if trackInputType == "name" then
```
Comment (roryc89) [inline src/MainUpdate.elm]: This is using strings where we make use of Elm Union Types

Comment (roryc89) [inline src/MainUpdate.elm]: I see this is addressed in the other PR :) Gonna review when it's merged

File: src/NewColumnView.elm [hunk 2]
```diff
@@ -25,24 +31,57 @@ view model =
...
 
+        disableInput columnId =
+            let
+                columnsWithSessions =
+                    model.datesWithSessions
+                        |> List.concatMap .sessions
+                        |> List.map .sessionColumn
+                        |> List.filter (\c -> c /= AllColumns)
+            in
+                if List.member (columnId) columnsWithSessions then
```
Comment (roryc89) [inline src/NewColumnView.elm]: no need for parens here

**Comment by roryc89** [comment]
Looks great. One tiny suggestion but it can easily be addressed in a future PR. Assigning nelson.

File: src/NewColumnView.elm (hunk 2 above)
Comment (naazy) [inline src/NewColumnView.elm]: Have removed in the next PR  😄

**Pull request merged**